### PR TITLE
Update readme and issue template for Pyrelight

### PR DESCRIPTION
**Pull request by out-of-phaze**
Updates the readme, issue template, pull request template, and changelog workflows.
Changelog workflows will not work until we have a bot account set up :(

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,53 @@
...
 </p>
 
+| Website                   | Link                                              |
+|---------------------------|---------------------------------------------------|
+| Code                      | [https://github.com/PyrelightSS13/Pyrelight](https://github.com/PyrelightSS13/Pyrelight)        |
+| Issues                    | [https://github.com/PyrelightSS13/Pyrelight/issues](https://github.com/PyrelightSS13/Pyrelight/issues) |
+| Upstream Issues           | [https://github.com/NebulaSS13/Nebula/issues](https://github.com/NebulaSS13/Nebula/issues)       |
+| Discord                   | Currently private, sorry!                         |
+| Nebula Discord            | [https://discord.gg/DYGXkN2S](https://discord.gg/DYGXkN2S)                       |
+| Coderbus Discord          | [https://discord.gg/Vh8TJp9](https://discord.gg/Vh8TJp9)                        |
```
Comment (MistakeNot4892) [inline README.md]: Why are we linking Coderbus specifically on our readme?

File: README.md [hunk 2]
```diff
@@ -1,38 +1,53 @@
...
+
+This is the codebase for Pyrelight, a middle-fantasy total conversion project for Space Station 13, derived from Nebula SS13. A round of this game intends to represent a lived day (or days) in the life of someone in a remote fantasy village, whether it's a day where they struck it rich or where their house burnt down and they caught the plague. An emphasis is placed on interconnected, generalized systems that allow for unique, emergent gameplay. Boil your tea with fire magic, forage herbs to concoct medicinal tinctures, or hunt in the woods and use your prey's bones and pelt to make your own equipment; what will *you* do?
```
Comment (MistakeNot4892) [inline README.md]: I would suggest something like this for readability and slightly more structure:
````
This is the codebase for Pyrelight, a middle-fantasy total conversion project for Space Station 13, derived from Nebula SS13. This fork has a focus on bringing together the Nebula planetary map mechanics with magic and fantasy-themed content and lore, to provide a unique twist on the general SS13 pattern.

A round of Pyrelight aims to represent a lived day (or days) in the life of someone in a remote fantasy village, whether it's a day where they struck it rich or where their house burnt down and they caught the plague. 

An emphasis is placed on interconnected, generalized systems that allow for unique, emergent gameplay. Boil your tea with fire magic, forage herbs to concoct medicinal tinctures, or hunt in the woods and use your prey's bones and pelt to make your own equipment; what will *you* do?
````
I am really not sold on the 'what will you do?' but I don't have anything else to end it with.

File: README.md (hunk 1 above)
Comment (out-of-phaze) [reply README.md]: because I stole the table layout from tg's readme

Comment (MistakeNot4892) [reply README.md]: I'm not going to block as I think this at least partially just my hangups about that community specifically, but I would prefer not to have it on our readme. I don't really like the association/implication in a way that I am struggling to put words around beyond 'I haven't had a good time in there'.

File: README.md (hunk 2 above)
Comment (MistakeNot4892) [reply README.md]: Just reiterating the above - I would at least like some paragraph formatting so it isn't one big block of text.

Comment (out-of-phaze) [reply README.md]: image
don't understand what you mean by paragraph formatting, what you put is the opposite of a paragraph, it's (nearly) each sentence on a separate line. it already has word-wrap as-is. i'll still split it up though

Comment (MistakeNot4892) [reply README.md]: They are quite long sentences dealing with different topics. Imo the combined block is less readable.

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**